### PR TITLE
Added Client Page

### PR DESCRIPTION
**Pull request by uzairmirza90**
Conflicts resolved with upstream master.

### CODE REVIEW

**Review by getalihassan — APPROVED** [review]

**Comment by getalihassan** [comment]
@umairwaheed @uzairmirza90 reviewed

**Comment by getalihassan** [comment]
@uzairmirza90 please resovlve conflicts

**Comment by uzairmirza90** [comment]
@getalihassan   Resolved.

File: components/ClientCard/ClientCard.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+const ClientCard: React.FC<ClientCardProps> = ({ image, name, project }) => {
+  return (
+    <Container maxWidth="md">
+      <Stack
+        direction={{ xs: 'column', md: 'row' }}
+        spacing={5}
+        my={3}
+        p={2}
+        sx={{
+          alignItems: 'center',
```
Comment (getalihassan) [inline components/ClientCard/ClientCard.tsx:20]: can we refactor this and do styling in MUI style?

**Review by getalihassan — CHANGES_REQUESTED** [review]

File: components/ClientCard/ClientCard.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          borderRadius: 4,
+        }}
+      >
+        <Box>
+          <Image src={image} width={150} height={150} objectFit='cover' />
+        </Box>
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
```
Comment (getalihassan) [inline components/ClientCard/ClientCard.tsx:31]: can we refactor this and do styling in MUI style?

File: styles/pagesStyles/clients.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
+.clients-page {
```
Comment (getalihassan) [inline styles/pagesStyles/clients.scss:1]: try not to use scss, use useStyle hook

**Review by getalihassan — CHANGES_REQUESTED** [review]

**Comment by uzairmirza90** [comment]
@getalihassan  Removed in-line styles into added into separate Scss file and conflicts are resolved.

**Review by getalihassan — APPROVED** [review]

**Comment by umairwaheed** [comment]
@uzairmirza90 can you make the CI pass?

**Review by umairwaheed — CHANGES_REQUESTED** [review]
CI is not passing.

**Review by umairwaheed — CHANGES_REQUESTED** [review]
@uzairmirza90 yarn.lock should be tracked. Don't delete it please. Can you also resolve conflicts?

**Review by umairwaheed — CHANGES_REQUESTED** [review]
There are conflicts.

**Comment by umairwaheed** [comment]
@uzairmirza90 can resolve the conflicts?

**Pull request merged**